src/main/java/dk/cngroup/lentils/entity/FinalPlace.java

```diff
@@ -0,0 +1,92 @@
+package dk.cngroup.lentils.entity;
+
+import org.springframework.data.geo.Point;
+
+import javax.persistence.*;
+import javax.validation.Constraint;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "finalPlace")
+public class FinalPlace {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private Long id;
+
+    @Size(min = 5, max = 50)
+    @Column(name = "placeTitle")
+    private String placeTitle;
+
+    @Column(name = "location")
+    private Point location;
+
+    @Column(name = "finalTime")
+    private LocalDateTime finalTime;
```

Comment:
Prejmenoval bych na "openningTime", protoze je to cas, kdy se zpristupni final place. Co myslis? FinalTime to uplne nevystihuje si myslim..